tools/mo/requirements_tf.txt

```diff
@@ -1,7 +1,6 @@
-# TensorFlow 1.x and 2.x are incompatible, use separate virtual environments for each version
-tensorflow~=1.15.5
+tensorflow~=2.5.0
 networkx~=2.5
-numpy>=1.16.6,<1.19
+numpy>=1.16.6,<1.20
 defusedxml>=0.7.1
 requests>=2.25.1
 fastjsonschema~=2.15.1
```

Comment:
please do not remove `requirements_tf2`. Let us just have two copies

Reply:
'requirements_tf2' file moved back